metron-deployment/packaging/docker/rpm-docker/SPECS/metron.spec

```diff
@@ -317,6 +316,8 @@ This package installs the Metron Profiler %{metron_home}
 # ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 %changelog
+* Thu Jan 19 2017 Justin Leet <justinjleet@gmail.com> - 0.3.0
```

Comment:
It is true that apache recommends strongly (or maybe outright forbids) things like author tags in source code (see [here](https://mail-archives.apache.org/mod_mbox/www-community/200306.mbox/%3C20030609234538.GA22335@lyra.org%3E) for a discussion, the reasoning was mostly around the author tags not being accurately representative.

For this situation, however, this is a changelog, so it doesn't have the problem of accurate representation that author tags do.  That being said, it is redundant information because such information is stored in git.

I'd recommend doing a dev list discussion with a subject that starts with [MENTORS] on this to see what the ASF wants.